multiregion/README.md

```diff
@@ -312,14 +329,50 @@ Sample output:
 
 Topic: multi-region-async	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
 	Topic: multi-region-async	Partition: 0	Leader: 3	Replicas: 2,1,3,4	Isr: 3,4	Offline: 2,1	Observers: 3,4
-...
+
+==> Describe topic multi-region-default
+
+Topic: multi-region-default	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
+	Topic: multi-region-default	Partition: 0	Leader: 3	Replicas: 2,1,3,4	Isr: 3,4	Offline: 2,1	Observers: 3,4
 ```
 
-Observations for topic `multi-region-async`:
+Observations for topics `multi-region-async` and `multi-region-default`:
 
 * It has a leader again (e.g. replica 3 in the above output)
 * The observers are now in the ISR list (e.g. replicas 3,4 in the above output)
 
+### Permanent Failover
+
+At this point in the example, if the brokers `broker-west-1` and `broker-west-2` come back online then by default the leaders for the topics `multi-region-async` and `multi-region-default` will be elected back to a replica with a rack of `west` (replica 1 or 2). To change this fallback behavior we can change the configuration of assignment of the affected topic. Here we are going to perform a permanent fallback to the east region for the topic `multi-region-default`.
```

Comment:
```suggestion
At this point in the example, if the brokers `broker-west-1` and `broker-west-2` come back online, then by default the leaders for the topics `multi-region-async` and `multi-region-default` will be elected back to a replica with a rack in `west` (i.e., replica 1 or 2). To change this fallback behavior we can change the configuration of assignment of the affected topic. Here we are going to perform a permanent fallback to the east region for the topic `multi-region-default`.
```

Reply:
What does "configuration of assignment of the affected topic" mean?

Reply:
IIUC, `fallback` is not the right term, I think it should be `failover`?

Reply:
Okay. I don't think `failover` is accurate either since it is not a failure. How about:

```
At this point in the example, if the brokers `broker-west-1` and `broker-west-2` come back online, then by default the leaders for the topics `multi-region-async` and `multi-region-default` will be elected back to a replica with a rack in `west` (i.e., replica 1 or 2). To change this behavior we can change the topic placement constraints configuration and replica assignment of the affected topic. Here we are going to change the topic placement constraints configuration and replica assignment for `multi-region-default`.
```